R/html_dependencies.R

```diff
@@ -1,4 +1,4 @@
-#' @import htmltools
+#' @import htmltools htmldeps
```

Comment:
I believe that since you are doing the explicit imports you don't need the `htmldeps` here as well

Reply:
Will remove, thanks!